_tips/developing.id.md

```diff
@@ -0,0 +1,638 @@
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Informasi Lebih Lanjut" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [Urutan yang Bermakna 1.3.2](/WAI/WCAG21/quickref/#meaningful-sequence) ([Memahami 1.3.2](/WAI/WCAG21/Understanding/meaningful-sequence))
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Tulis kode yang mampu menyesuaikan dengan teknologi pengguna
+
+Gunakan desain responsif untuk menyesuaikan tampilan dengan kondisi zum dan ukuran porta pandang yang berbeda, seperti pada perangkat seluler dan tablet. Jika ukuran fon diperbesar setidaknya 200%, hindari pengguliran layar secara horizontal dan cegah adanya konten yang terpotong. Gunakan metode Peningkatan Progresif untuk membantu memastikan fungsionalitas inti dan konten tersedia terlepas dari teknologi yang digunakan.
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Contoh: Menggunakan kueri media untuk menyesuaikan navigasi" class="example" %}
+
+<div class="adapt-code">
+  <figure>
+    <div class="code">
+      <div class="grid-4">
+        <div class="from-col1 to-col2">
+{:/}
+
+~~~css
+/* Pada porta pandang sempit, gunakan navigasi lebar penuh */
+@media screen and (min-width: 25em) {
+  #nav {
+    float: none;
+    width: auto;
+  }
+  #main {
+    margin-left: 0;
+  }
+}
+~~~
+
+{::nomarkdown}
+</div>
+<div class="from-col3 to-col4">
+{:/}
+
+~~~css
+/* Pada porta pandang lebar, letakkan navigasi di sebelah kiri */
+@media screen and (min-width: 43em) {
+  #nav {
+    float: left;
+    width: 24%;
+  }
+  #main {
+    margin-left: 27%;
+  }
+}
+~~~
+
+{::nomarkdown}
+      </div>
+    </div>
+  </div>
+  </figure>
+</div>
+
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Informasi Lebih Lanjut" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [Mengubah ukuran teks 1.4.4](/WAI/WCAG21/quickref/#resize-text) ([Memahami 1.4.4](/WAI/WCAG21/Understanding/resize-text))
+  * [Identifikasi yang Konsisten 3.2.4](/WAI/WCAG21/quickref/#consistent-identification) ([Memahami 3.2.4](/WAI/WCAG21/Understanding/consistent-identification))
+* **Latar Belakang**
+  * [Layar Ukuran Kecil](/TR/mobile-accessibility-mapping/#h-small-screen-size)
+* **Kisah Pengguna**
+  * [Menjelaskan bagaimana alternatif tampilan pada halaman yang dizum dapat membantu](/people-use-web/user-stories/#retiree)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Sediakan makna untuk elemen interaktif yang nonstandar
+
+Gunakan <abbr title="Aplikasi Internet yang Kaya dan Aksesibel">WAI-ARIA</abbr> untuk memberikan informasi tentang fungsi dan status *widget* yang dirancang khusus, seperti akordeon dan tombol yang dirancang khusus. Misalnya, `role="navigation"` dan `aria-expanded="true"`. Kode tambahan diperlukan untuk mengimplementasikan perilaku *widget* tersebut, seperti memperluas dan menyusutkan konten, atau bagaimana *widget* merespons aksi dari kibor.
```

Comment:
@frirasyidi Please wrap "widget" in `<span lang="en"></span>`.

Reply:
@remibetin done in the next commit.